client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -363,6 +363,7 @@ public void appendSetup(AppendSetup appendSetup) {
                     if (e == null) {
                         state.connectionSetupComplete(connection);
                     } else {
+                        log.debug("Error while sending inflight during setup append for writer " + writerId, e);
```

Comment:
I think this is just going to double log the exception. failConnection should already be logging.

Reply:
removed it.